modules/nw-operator-cr.adoc

```diff
@@ -23,239 +23,300 @@ endif::[]
 [id="nw-operator-cr_{context}"]
 = Cluster Network Operator configuration
 
-The configuration for the cluster network is specified as part of the Cluster Network Operator (CNO) configuration and stored in a custom resource (CR) object that is named `cluster`. The CR specifies the parameters for the `Network` API in the `operator.openshift.io` API group.
+The configuration for the cluster network is specified as part of the Cluster Network Operator (CNO) configuration and stored in a custom resource (CR) object that is named `cluster`. The CR specifies the fields for the `Network` API in the `operator.openshift.io` API group.
 
-ifdef::post-install-network-configuration[]
+The CNO configuration inherits the following fields during cluster installation from the `Network` API in the `Network.config.openshift.io` API group and these fields cannot be changed:
+
+- `clusterNetwork`
+- `serviceNetwork`
+- `defaultNetwork.type`
+
+// For the post installation assembly, no further content is provided.
+ifdef::post-install-network-configuration,operator[]
 [NOTE]
 ====
 After cluster installation, you cannot modify the configuration for the cluster network provider.
 ====
-endif::post-install-network-configuration[]
+endif::[]
 ifndef::post-install-network-configuration[]
-You can specify the cluster network configuration for your {product-title} cluster by setting the parameter values for the `defaultNetwork` parameter in the CNO CR. The following CR displays the default configuration for the CNO and explains both the parameters you can configure and the valid parameter values:
+You can specify the cluster network provider configuration for your cluster by setting the fields for the `defaultNetwork` object in the CNO object named `cluster`.
+
+[id="nw-operator-cr-cno-object_{context}"]
+== Cluster Network Operator configuration object
+
+The fields for the Cluster Network Operator (CNO) are described in the following table:
+
+.Cluster Network Operator configuration object
+[cols=".^3,.^2,.^5a",options="header"]
+|====
+|Field|Type|Description
+
+|`metadata.name`
+|`string`
+|The name of the CNO object. This name is always `cluster`.
+
+|`spec.clusterNetwork`
+|`array`
+|A list specifying the blocks of IP addresses from which pod IP addresses are
+allocated and the subnet prefix length assigned to each individual node. For example:
 
-.Cluster Network Operator custom resource
 [source,yaml]
-ifndef::operator[]
 ----
-apiVersion: operator.openshift.io/v1
-kind: Network
-metadata:
-  name: cluster
 spec:
-  clusterNetwork: <1>
-  - cidr: 10.128.0.0/14
+  clusterNetwork:
+  - cidr: 10.128.0.0/19
+    hostPrefix: 23
+  - cidr: 10.128.32.0/19
     hostPrefix: 23
```

Comment:
for some reason, in the preview, the first `23` is green but the second `23` is red...

(It's a little confusing that this is being discussed on the "Installing a cluster on AWS with network customizations" page, since this is only relevant _after_ installation...)

Reply:
@danwinship, which part? `clusterNetwork` and `serviceNetwork`?

Reply:
Yes; if you want to change those at install time, you have to set them in `install-config`, not here.

Maybe on the "Installing" page it should just clarify that they should be considered read-only.